mpisppy/spbase.py

```diff
@@ -86,7 +86,8 @@ def __init__(
         self.n_proc = self.mpicomm.Get_size()
         self.global_rank = MPI.COMM_WORLD.Get_rank()
 
-        global_toc("Initializing SPBase")
+        if not options.get("no_toc", False):
```

Comment:
```suggestion
        if options.get("toc", True):
```
I don't really like negative options ...

Reply:
I do... but I'll let you win this one! (You are almost surely in the majority)